lib/deck.rb

```diff
@@ -0,0 +1,37 @@
+class Deck
+  attr_reader :cards, :stem_cards, :geography_cards, :turing_cards, :pop_cards
+  def initialize(cards)
+    @cards = cards
+    @stem_cards = []
+    @geography_cards = []
+    @turing_cards = []
+    @pop_cards = []
+    cards.each do |card|
+
+    if card.category == :STEM
+      @stem_cards << card
+    elsif card.category == :Geography
+      @geography_cards << card
+    elsif card.category == :Turing
+      @turing_cards << card
+    elsif card.category == :Pop
```

Comment:
The last statement of any `if/elsif` statement can more often than not just be an `else`

```ruby
if card.category == :STEM
  @stem_cards << card
elsif card.category == :Geography
  @geography_cards << card
elsif card.category == :Turing
  @turing_cards << card
else #<=== If it isn't one of the other options then naturally it has to be this option by default otherwise you risk not having any of the options returning true and possibly getting some unexpected behavior down the line somewhere
  @pop_cards << card
end
```